src/fparser/scripts/fparser2.py

```diff
@@ -63,38 +64,50 @@
 # OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
 # DAMAGE.
 
-from fparser.scripts.script_options import set_f2003_options
+''' Example script to parse a Fortran program using fparser '''
+from __future__ import print_function
+from fparser.scripts.script_options import set_fparser_options
```

Comment:
If I pip install this branch of fparser and then go to src/fparser/scripts:

    (py2) [kbc59144@ijb87vig scripts]$ ./fparser2.py 
    Traceback (most recent call last):
      File "./fparser2.py", line 69, in <module>
        from fparser.scripts.script_options import set_fparser_options
    ImportError: No module named scripts.script_options

The directory src/fparser/scripts contains no `__init__.py` and therefore is not a part of the fparser package. We could solve this for now by just using a relative import:

    from script_options import set_fparser_options

Reply:
Ah. It was working for me as I'd added an __init__.py file manually but not added it to the repo. I've gone for that option instead.